src/website/app/GuidelinePage.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { createStyledComponent } from '../../utils';
+
+const Root = createStyledComponent('div', ({ theme }) => ({
+  '& > div > h1 ~ p': {
+    fontSize: theme.fontSize_h3,
+    lineHeight: theme.lineHeight_prose,
```

Comment:
We're already styling all markdown content with this line-height (see Markdown.js).